docs/chapter_computational_complexity/space_time_tradeoff.md

```diff
@@ -69,11 +87,26 @@
 === "C++"
 
     ```cpp title="leetcode_two_sum.cpp"
-
+    
     ```
 
 === "Python"
 
     ```python title="leetcode_two_sum.py"
     
     ```
+
+=== "Go"
+
+    ```go title="leetcode_two_sum.go"
+    func twoSumHashTable(nums []int, target int) []int {
```

Comment:
We prefer `space` to `tab` (equivalent to 4 spaces) to avoid strange bugs.

Reply:
`tab` will be replaced with 4 spaces in the next patch

Reply:
In Golang, Tabs for indentation is the official recommendation, Docker and Kubernets also use tabs. The correctness of the code is guaranteed by gofmt. More in https://go.dev/doc/effective_go#formatting

Reply:
> `tab` will be replaced with 4 spaces in the next patch

This comment is deprecated.

Reply:
Thanks for the clear description.